docs/admin/install.rst

```diff
@@ -399,38 +407,36 @@ MySQL and MariaDB
 
 .. hint::
 
-    Some Weblate features will perform better with :ref:`postgresql`. This
-    includes searching and translation memory, which both utilize full-text
+    Some Weblate features will perform better with :ref:`postgresql`.
+    This includes searching and translation memory, which both utilize full-text
     features in the database and PostgreSQL implementation is superior.
 
-Weblate can be also used with MySQL or MariaDB, please see
-:ref:`django:mysql-notes` and :ref:`django:mariadb-notes` for caveats using
-Django with those. Because of the limitations it is recommended to use
+Weblate can also be used with MariaDB (10.2.7 or later) or MySQL (5.7.8 or later),
+please read :ref:`django:mysql-notes` and :ref:`django:mariadb-notes` for caveats using
+Django with those. Because of these limitations it is recommended to use
 :ref:`postgresql` for new installations.
 
-Weblate requires MySQL at least 5.7.8 or MariaDB at least 10.2.7.
-
-Following configuration is recommended for Weblate:
+The following configuration is recommended for Weblate:
 
 * Use the ``utf8mb4`` charset to allow representation of higher Unicode planes (for example emojis).
 * Configure the server with ``innodb_large_prefix`` to allow longer indices on text fields.
 * Set the isolation level to ``READ COMMITTED``.
 * The SQL mode should be set to ``STRICT_TRANS_TABLES``.
 
 MySQL 8.x, MariaDB 10.5.x or newer have reasonable default configuration so
-that no server tweaking should be necessary and all what is needed can be
+that no server tweaking should be necessary, and all that is needed can be
 configured on the client side.
 
 Below is an example :file:`/etc/my.cnf.d/server.cnf` for a server with 8 GB of
-RAM. These settings should be sufficient for most installs. MySQL and MariaDB
-have tunables that will increase the performance of your server that are
+RAM. These settings should be sufficient for most installs.
+MySQL and MariaDB have tunables to increase server performance that are
 considered not necessary unless you are planning on having large numbers of
-concurrent users accessing the system. See the various vendors documentation on
-those details.
+concurrent users accessing the system. See the various vendors documentation for
+details about this.
 
-It is absolutely critical to reduce issues when installing that the setting
-``innodb_file_per_table`` is set properly and MySQL/MariaDB restarted before
-you start your Weblate install.
+To reduce issues it is absolutely critical that the
+``innodb_file_per_table`` setting is configured correctly and that
```

Comment:
MOVE. Could possibly be that a script calls for these to be restarted to see that they work.